services/orbits/orbits/schemas.go

```diff
@@ -113,80 +124,93 @@ var idealOrbitSchema = map[string]*schema.Schema{
-        ValidateFunc: validation.IsUUID,
-    },
-    "metric_id_for_velocity_x": {
-        Type:         schema.TypeString,
-        Required:     true,
-        ValidateFunc: validation.IsUUID,
-    },
-    "metric_id_for_velocity_y": {
-        Type:         schema.TypeString,
-        Required:     true,
-        ValidateFunc: validation.IsUUID,
-    },
-    "metric_id_for_velocity_z": {
-        Type:         schema.TypeString,
-        Required:     true,
-        ValidateFunc: validation.IsUUID,
-    },
+	"metric_id_for_position_x": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"metric_id_for_position_y": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"metric_id_for_position_z": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"metric_id_for_velocity_x": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"metric_id_for_velocity_y": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
+	"metric_id_for_velocity_z": {
+		Type:         schema.TypeString,
+		Required:     true,
+		ValidateFunc: validation.IsUUID,
+	},
 }
 
 var standardDeviationsSchema = map[string]*schema.Schema{
-    "latitude": {
-        Type:         schema.TypeFloat,
-        Required:     true,
-        ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, 1.0),
-    },
-    "longitude": {
-        Type:         schema.TypeFloat,
-        Required:     true,
-        ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, 1.0),
-    },
-    "altitude": {
-        Type:         schema.TypeFloat,
-        Required:     true,
-        ValidateFunc: helper.FloatAtLeastAndLessThan(0.0, 5000.1),
-    },
-    "ground_speed": {
-        Type:         schema.TypeFloat,
-        Required:     true,
-        ValidateFunc: helper.FloatAtLeastAndLessThan(0.0, 5000.1),
-    },
+	"latitude": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, 1.0),
+	},
+	"longitude": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, 1.0),
+	},
+	"altitude": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.0, 5000.1),
+	},
+	"ground_speed": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.0, 5000.1),
+	},
+}
+
+var satelliteConfigurationSchema = map[string]*schema.Schema{
+	"drag_cross_section": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, math.Inf(1)),
+	},
+	"radiation_cross_section": {
+		Type:         schema.TypeFloat,
+		Required:     true,
+		ValidateFunc: helper.FloatAtLeastAndLessThan(0.1, math.Inf(1)),
```

Comment:
I think there is some existing validation method to simply check FloatAtLeast, isn't there?